pkg/version/version.go

```diff
@@ -11,10 +11,6 @@ var (
 	buildDate  = "none"
 )
 
-const (
```

Comment:
You should move the modification of `version.go` to another PR, it's the different function